meta/main.yml

```diff
@@ -7,7 +7,7 @@ galaxy_info:
   description: Jenkins CI
   company: "Midwestern Mac, LLC"
   license: "license (BSD, MIT)"
-  min_ansible_version: 2.0
+  min_ansible_version: 2.2
```

Comment:
Please keep this 2.0 for now and change the usages to `always_run: yes` instead (at least for the short term).